src/modes/top10/Top10SortHint.svelte

```diff
@@ -8,14 +8,15 @@
 
 <style>
   .root {
-    display: flex;
     cursor: pointer;
-    text-align: center;
-    justify-content: center;
     position: relative;
   }
 
+  .hint-wrapper {
```

Comment:
please fix the warning posted by github action

Reply:
done